pandas/tests/frame/test_sorting.py

```diff
@@ -269,6 +285,11 @@ def test_sort_datetimes(self):
         df2 = df.sort_values(by=['B'])
         assert_frame_equal(df1, df2)
 
```

Comment:
This particular change does not have anything to do with the fix. I added it to fix the test since I saw that its goal was broken